src/main/kotlin/com/re/back/enums/UserRole.kt

```diff
@@ -0,0 +1,5 @@
+package com.re.back.enums
+
+enum class UserRole {
+    ROLE_USER
```

Comment:
What about USER?

Reply:
That's  necessary in Spring Security context, I guess there is internal mechanism to determine the ROLE of the user if you defined specific role for specific endpoint, that allows this naming convention : `ROLE_{role_name}`